dialogue-core/src/main/java/com/palantir/dialogue/core/DialogueChannel.java

```diff
@@ -49,6 +49,7 @@
 public final class DialogueChannel implements Channel {
     private final Map<String, LimitedChannel> limitedChannelByUri = new ConcurrentHashMap<>();
     private final AtomicReference<LimitedChannel> nodeSelectionStrategy = new AtomicReference<>();
+    private final QueuedChannel queuedChannel; // just so we can process the queue when uris reload
```

Comment:
Keeping this field around might tempt people to call the `QueuedChannel#maybeExecute` method directly instead of sending everything down through the fully decorated `delegate`. I could just keep around an interface that literally just has the `schedule()` method on it to prevent this... thoughts?

Reply:
The runnable idea protects us from misuse, but I think the abstraction makes the code less clear. imo this is fine as is.